### PR TITLE
[ADD] switch_noupdate

### PR DESCRIPTION
**Pull request by mreficent**
To be able to switch noupdate easily in several records.

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -2258,3 +2259,16 @@ def chunked(records, single=True):
...
+
+def switch_noupdate(env, xml_ids, value):
+    for xml_id in xml_ids:
+        if '.' not in xml_id:
+            logger.error(
+                'Need the module reference to be specified in the IDs' % xml_id)
+        else:
+            env.cr.execute("""
+                UPDATE ir_model_data
+                SET noupdate = NOT COALESCE(noupdate, FALSE)
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: Must be `value`

Comment (mreficent) [reply openupgradelib/openupgrade.py]: Then the name of the method is wrong. "switch" means change the noupdate **always**. If you want a method that only puts the passed value, then the name "switch" is wrong, as sometimes you only want to assure a value but you really are not changing it 😏

File: openupgradelib/openupgrade.py [hunk 2]
```diff
@@ -2258,3 +2259,16 @@ def chunked(records, single=True):
                 yield record
             continue
         yield chunk
+
+
+def set_noupdate_value(env, xml_ids, value):
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]:
```suggestion
def set_xmlid_noupdate_value(env, xml_ids, value):
```

File: openupgradelib/openupgrade.py [hunk 3]
```diff
@@ -2258,3 +2259,20 @@ def chunked(records, single=True):
...
+    """Set the xml_ids noupdate values in a module.
+
+    :param module: module name
+    :param xml_ids: a tuple or list of xml record IDs
+    :param bool value: True or False.
+    """
+    if not isinstance(xml_ids, (list, tuple)):
+        do_raise("XML IDs %s must be a tuple or list!" % xml_ids)
+
+    env.cr.execute("""
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade.py]: openupgrade.logged_query

**Pull request merged**